stmt/stmts/insert.go

```diff
@@ -185,15 +185,44 @@ func (s *InsertIntoStmt) getColumns(tableCols []*column.Col) ([]*column.Col, err
 	return cols, nil
 }
 
+func (s *InsertIntoStmt) getDefaultValues(ctx context.Context, cols []*column.Col) (map[interface{}]interface{}, error) {
+	m := map[interface{}]interface{}{}
+	for _, v := range cols {
+		if value, ok, err := getDefaultValue(ctx, v); ok {
+			if err != nil {
+				return nil, errors.Trace(err)
+			}
+
+			m[v.Name.L] = value
+		}
+	}
+
+	return m, nil
+}
+
+func (s *InsertIntoStmt) getSetlist() error {
```

Comment:
fillValueList is a better name.
"getSetList" only returns an error is a little weird. This function fills s.Lists with s.Setlist.